src/modules/CrossReferences.jl

```diff
@@ -134,18 +136,8 @@ end
 # Issues/PRs cross referencing.
 # -----------------------------
 
-if isdefined(Base, :LibGit2)
-    function issue_xref(link::Markdown.Link, num, meta, page, doc)
-        remote =
-            LibGit2.with(LibGit2.GitRepoExt(dirname(doc.user.root))) do repo
-                LibGit2.with(LibGit2.GitConfig(repo)) do cfg
-                    Utilities.getremote(cfg)
-                end
-            end
-        isnull(remote) || (link.url = "https://github.com/$(get(remote))/issues/$(num[2:end])")
-    end
-else
-    issue_xref(link::Markdown.Link, num, meta, page, doc) = nothing
+function issue_xref(link::Markdown.Link, num, meta, page, doc)
+    link.url = "https://github.com/$(doc.internal.remote)/issues/$num"
```

Comment:
Not very relevant for this PR, but should we have a `github_url` or something argument for mkdocs to determine this? The remote may not be 100% reliable, e.g. if you're building on a cloned fork. We should default to origin remote though, I think.